src/pages/about/index.tsx

```diff
@@ -0,0 +1,35 @@
+import { Box } from "@nypl/design-system-react-components";
+import Header from "@/components/header/header";
+import NotificationBanner from "@/components/notificationBanner/notificationBanner";
+import aboutData from "@/data/aboutData";
+
+export default function About() {
+  function createSection(heading: React.JSX.Element, body: React.JSX.Element) {
+    return (
+      <>
+        {heading}
+        {body}
+      </>
+    );
+  }
+
+  return (
+    <>
+      <NotificationBanner />
+      <Header />
+      <Box
+        sx={{
+          maxWidth: "820px",
+          margin: "s",
+          [`@media screen and (min-width: 820px)`]: {
```

Comment:
Would using the `TemplateAppContainer` component like the home page help with the static width value? The About page should follow the standard width and now I'm realizing I did not check that detail in the designs before.

Reply:
Yea this was weird. I was assuming (but obviously did not check when reviewing the design 🤦‍♀️) that if the header was on new breakpoints, the width of the content would align. This `820px` width content is in a weird spot between the small tablet breakpoint (`768px`) and the old `lg` breakpoint (`960px`). Bc it's not either, and the page is just this straight up static content, I'm not using the new breakpoints at all as you can see here– all that matters, to create the breakpoint behavior as it is in Figma, is whether the screen width is larger or smaller than the content. Which is all a very long way to say are we gonna keep the content this width knowing that it's not "standard" ?

Reply:
"820px" is directly from Figma, right? We should point it out to Lily and mention we can keep it but it's not standard and this might change when the new breakpoints are in place globally (and not just in the Header).

This _feels_ off to me but this is up to Lily to test and decide.